packages/create-plugin/src/commands/generate.command.ts

```diff
@@ -69,6 +69,8 @@ function getTemplateData(answers: CliArgs) {
     isNPM: packageManagerName === 'npm',
     version: currentVersion,
     bundleGrafanaUI: features.bundleGrafanaUI,
+    useReactRouterV6: features.useReactRouterV6 && pluginType === PLUGIN_TYPES.app,
+    reactRouterVersion: features.useReactRouterV6 && pluginType === PLUGIN_TYPES.app ? '6.22.0' : '5.2.0',
```

Comment:
can we make this work with only the boolean parameter? as they are right now it seems this second one is just a calculated field from the previous one.

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/710/commits/17aff14e5abbfc595ce038235fc0d8fa57a3aa0d